lib/livebook/config.ex

```diff
@@ -228,6 +236,14 @@ defmodule Livebook.Config do
     end
   end
 
+  @doc """
+  Parses and validates the base url path from env.
+  Remove trailing '/' if present.
```

Comment:
We can check and raise an error :)

```suggestion
```

Reply:
Added method to remove it if present

Reply:
Sounds good, I would also remove this doc line :)

Btw. you can do `git checkout main static` to revert the static/ diff.

Reply:
I misunderstood for the static folder, no problem!